openedx/features/course_experience/README.rst

```diff
@@ -0,0 +1,54 @@
+Course Experience
+-----------------
+
+This directory contains a Django application that provides the overall
+experience for navigating around an Open edX course.
```

Comment:
I think the name `course_experience` and this description over-generalizes what this feature is really about.  We can discuss which is more correct, but I'd start with this description.  Possible replacement text:
```
This directory contains a Django application that provides the Course Home page (or course landing page), and various resources in support of the landing experience.
```